html/modules/custom/reliefweb_job_tagger/reliefweb_job_tagger.module

```diff
@@ -261,6 +289,44 @@ function reliefweb_job_tagger_node_presave(EntityInterface $node) {
   }
 }
 
+/**
+ * Implements hook_ENTITY_TYPE_view().
+ */
+function reliefweb_job_tagger_node_view(array &$build, EntityInterface $entity, EntityViewDisplayInterface $display, $view_mode) {
```

Comment:
I think it would be cleaner to add a local task tab like the "Delete" one with the button to requeue the job instead of adding it to the main job view mode.

Reply:
Done